### PR TITLE
[leveragelp]: fix new add collateral flow 

### PR DESCRIPTION
**Pull request by cryptokage1996**

### CODE REVIEW

**Review by Wgil — CHANGES_REQUESTED** [review]
We need tests that confirm the use case we're working with. User should be able to add collateral by going through the consolidate flow.

File: x/leveragelp/types/msgs.go [hunk 1]
```diff
@@ -111,7 +111,7 @@ func (msg *MsgOpen) ValidateBasic() error {
 		return errorsmod.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
 	}
 	// leverage should be greater than 1
```
Comment (Wgil) [inline x/leveragelp/types/msgs.go:113]: This comment doesn't make sense after this update

**Comment by Wgil** [comment]
@avkr003 would you suggest any other test to be created that confirms this use case?

**Comment by cosmic-vagabond** [comment]
@cryptokage1996 could you query the newly created position id `267440` and share the result of the query here thanks

**Pull request merged**